dev/release/verify-release-candidate.sh

```diff
@@ -656,7 +656,7 @@ test_linux_wheels() {
 }
 
 test_macos_wheels() {
-  local py_arches="3.5m 3.6m 3.7m 3.8"
+  local py_arches="3.5m 3.6m 3.7m 3.8 3.9"
```

Comment:
Could you revert this and other macOS wheel support related changes?
Let's work on macOS wheel support in separated pull request.